lib/susedistribution.pm

```diff
@@ -680,7 +686,7 @@ sub activate_console {
         $self->set_standard_prompt($user, skip_set_standard_prompt => $args{skip_set_standard_prompt});
         assert_screen $console;
     }
-    elsif ($type eq 'virtio-terminal') {
+    elsif ($type =~ /virtio-terminal|sut-serial/) {
```

Comment:
May not change anything, but all other type checks are using full word checking so I would do it here as well, to avoid unexpected matches.
```suggestion
    elsif ($type =~ /^virtio-terminal|sut-serial$/) {
```

Reply:
Good point, Clements, thanks!

Reply:
Matched as `elsif ($type =~ /^virtio-terminal|sut-serial$/) {`